src/sentry/utils/performance_issues/detectors/consecutive_http_detector.py

```diff
@@ -128,8 +143,42 @@ def _is_eligible_http_span(self, span: Span) -> bool:
         if any([x in description for x in ["_next/static/", "_next/data/"]]):
             return False
 
+        # If the event is from a JS SDK and the span ends after the LCP, we ignore it
+        # because it won't be a candidate for improving LCP
+        if self._is_event_from_browser_javascript_sdk() and self._span_occurs_after_lcp(span):
+            return False
+
         return True
 
+    def _span_occurs_after_lcp(self, span):
+        return (
+            datetime.fromtimestamp(span.get("timestamp"))
+            > datetime.fromtimestamp(self._event.get("start_timestamp"))
+            + timedelta(milliseconds=self.lcp)
+            if self.lcp is not None
+            else True
+        )
+
+    def _is_event_from_browser_javascript_sdk(self):
```

Comment:
It doesn't seem like we have a general util for checking if the event came from a frontend SDK, but I found [this in the frontend code](https://github.com/getsentry/sentry/blob/master/static/app/components/events/interfaces/spans/utils.tsx#L486) that I pulled into here